src/features/gestion-des-cookies/pages/gestion-des-cookies/gestion-des-cookies.page.html

```diff
@@ -0,0 +1,50 @@
+<div class="container m-5">
+  <h1 class="my-5 mx-5">Gestion des cookies</h1>
+  <h3 class="my-5 mx-5 text-primary">Cookies</h3>
+  <div class="row">
+    <div class="col-7 mx-5">
+      <p>
+        Ce site dépose un petit fichier texte (un « cookie ») sur votre ordinateur lorsque vous le consultez. Cela nous permet
+        de mesurer le nombre de visites et de comprendre quelles sont les pages les plus consultées.
+      </p>
+      <p>
+        Nous n'affichons pas de banière de consentement aux cookies car nos outils de suivi d'audience sont correctement
+        configurés pour respecter la vie privée des utilisateurs. Certains cookies sont dispensés du recueil préalable de votre
+        consentement dans la mesure où ils sont strictement nécessaires à la fourniture du service. Les traceurs ont vocation à
+        être conservés sur le poste informatique de l'internaute pour une durée allant jusqu'à 13 mois.
+      </p>
+    </div>
+  </div>
+</div>
+<div class="container m-5">
+  <h3 class="my-5 mx-5 text-primary">Mesure d’audience</h3>
+  <div class="row">
+    <div class="col-7 mx-5">
+      <p>
+        Certains cookies permettent d’établir des mesures statistiques de fréquentation et d’utilisation du site pouvant être
+        utilisées à des fins de suivi et d’amélioration du service :
+      </p>
+      <ul>
+        <li>Les données collectées ne sont pas recoupées avec d’autres traitements.</li>
+        <li>Le cookie déposé sert uniquement à la production de statistiques anonymes.</li>
+        <li>Le cookie ne permet pas de suivre la navigation de l’internaute sur d’autres sites.</li>
+      </ul>
+      <p>
+        La mesure d’audience (nombre de visites, pages consultées) est réalisée via un outil libre intitulé
+        <a href="https://matomo.org/" target="_blank">Matomo</a>
```

Comment:
```suggestion
        <a href="https://matomo.org/" target="_blank" rel="noopener noreferrer">Matomo</a>
```
https://stackoverflow.com/questions/40593632/use-window-open-but-block-use-of-window-opener